modules/cco-short-term-creds-auth-flow-aws.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing_cloud_provider_credentials/cco-short-term-creds.adoc
+
+:_content-type: REFERENCE
+[id="cco-short-term-creds-auth-flow-aws_{context}"]
+= AWS Security Token Service authentication process
+
```

Comment:
Since this is still a module, it should probably have some intro content

(Also for accessibility reasons it'd also be good to describe the info in the graphic too)

Same comment for the other modules w/ graphics too

Reply:
Will add a brief module intro for each, and work on getting proper image descriptions in later (target: having all three described when the real Azure diagram is ready to add)